src/tokens/fontTokens.stories.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+  import { Meta, Story, Template } from '@storybook/addon-svelte-csf'
+  import { font as allFonts } from '../../tokens/css'
+  import FontTokenSwatchGroup from '../storyHelpers/FontTokenSwatchGroup.svelte'
+
+  const { macos, monospace, windows, primary, secondary, mobile, components } =
+    allFonts
+</script>
+
+<Meta title="Font Tokens" />
+
+<Story name="All Fonts">
+  <FontTokenSwatchGroup tokens={allFonts} />
```

Comment:
Similar, I'd be tempted to either only have the `All Fonts` or only the sub groups

Reply:
I'd personally support both since there are some tokens (like white and black) that don't actually fall into a category (and are therefore caught by `All`), and because it's nice to drill into a specific category like `Primary` for fonts or `Primitive` for colors.